### PR TITLE
Avoid calling is_active too early and add a warning message

### PR DESCRIPTION
**Pull request by zzuga**
## Can you merge #7 first please

At this point, the gravityforms plugin may have been loaded or not.

For shoot it works but for gcse it doesnt, because plugins are loaded in alphabetical order.

I added a call to _doing_it_wrong() so we are alerted about this during development

### CODE REVIEW

**Comment by adamf321** [comment]
This solution looks wrong to me - we should call is_active where it was before. Have you tried just running the init() function on the 'init' action? That should run after all plugins have been loaded.

**Comment by zzuga** [comment]
Keep in mind that we are only adding filters inside that if.

If gravityforms isnt installed, nothing will happen. It should be ok.

If we were really interacting with GF it would be different, but we are not.

My concern is that moving to 'init' might have side effects.
Let me know your thoughts.

File: src/Actions/Signup.php [hunk 1]
```diff
@@ -147,10 +147,11 @@ private static function generate_username( $args ) {
 		if ( isset( $args['display_name'] ) ) {
 			$username = $args['display_name'];
 		} else {
-			$username =
+			$username = trim(
```
Comment (zzuga) [inline src/Actions/Signup.php:150]: This change comes from #7 
I incorrectly merged into this branch instead of master.

**Comment by zzuga** [comment]
Im calling init() the same way we call all other inits (in Bootstrap::init).
Actions\Signup::init( $form_id );
https://github.com/moxienyc/gcse-back-end/blob/GCSE-74/wp-content/plugins/gcse/src/Modules/Users/Bootstrap.php#L23

@adamf321 Are you suggesting I change this?
In that case, should we still keep the _doing_it_wrong call?

**Comment by zzuga** [comment]
@adamf321 Does this solution look better?
I tested and it works fine too.

**Pull request merged**